internal/dkg/state_machine.go

```diff
@@ -20,17 +20,42 @@ import (
 type Status uint32
 
 const (
+	// Fresh is the state all nodes start in - both pre-genesis, and if the network is running but they aren't
+	// yet participating
 	Fresh Status = iota
+	// Proposed implies somebody else has sent me a proposal
 	Proposed
+	// Proposing implies I have sent the others in the network a proposal
 	Proposing
+	// Accepted means I have accepted a proposal received from somebody else
+	// note Joiners do not accept/reject proposals
 	Accepted
+	// Rejected means I have rejected a proposal received from somebody else
+	// it doesn't automatically abort the DKG, but the leader is advised to abort and suggest some new terms
 	Rejected
+	// Aborted means the leader has told the network to abort the proposal; a node may have rejected,
+	// they may have found an error in the proposal, or any other reason could have occurred
 	Aborted
+	// Executing means the leader has reviewed accepts/rejects and decided to go ahead with the DKG
+	// this implies that the Kyber DKG process has been started
 	Executing
+	// Complete means the DKG has finished and a new group file has been created successfully
 	Complete
+	// TimedOut means the proposal timeout has been reached without entering the `Executing` state
+	// any node can trigger this for themselves should they identify timeout has been reached
+	// it does _not_ guarantee that other nodes have also timed out - a network partition or something else
+	// could have occurred. If the rest of the network continues, our node will likely transition to `Evicted`
 	TimedOut
+	// Joined is the state a new proposed group member enters when they have been proposed a DKG and they run the
+	// `join` DKG command to signal their acceptance to join the network
 	Joined
```

Comment:
there are two reasons:
- joiners follow a different code path than acceptor/rejectors. They could be combined, but I think it hurts reading of the code a little when they do.
- conceptually somebody who hasn't been accepted into the network doesn't have a say in its running. If we wanted to do automation around acceptance it might be relevant, although it's not right now